### PR TITLE
PR: Update FontAwesome and Material design icons

### LINKED ISSUE #119: Update FontAwesome and Material design icons
@steff456, please follow the instructions in UPDATE.md (present in the root of this repo) to update both fonts. Thanks!

### PR DESCRIPTION
**Pull request by steff456**
Fixes #119

### CODE REVIEW

**Comment by ccordoba12** [comment]
> Why did you remove fontawesome5-brands-webfont.ttf?

Sorry, I just saw that you didn't remove it. Please forget about this comment.

**Comment by ccordoba12** [comment]
Great job @steff456! One last thing: please update the link to the Material design version you updated to in our Readme:

https://github.com/spyder-ide/qtawesome/blame/master/README.md#L64

File: README.md [hunk 1]
```diff
@@ -51,17 +51,17 @@ The following prefixes are currently available to use:
...
+    - `fa5s` prefix has [935 icons in the "**solid**" style.](https://fontawesome.com/icons?d=gallery&s=solid&v=5.0.0,5.0.1,5.0.10,5.0.11,5.0.12,5.0.13,5.0.2,5.0.3,5.0.4,5.0.5,5.0.6,5.0.7,5.0.8,5.0.9,5.1.0,5.1.1,5.2.0,5.3.0,5.3.1,5.4.0,5.4.1,5.4.2,5.5.0,5.6.0,5.6.1,5.6.3,5.9.0&m=free)
+    - `fa5b` prefix has [413 icons of various **brands**.](https://fontawesome.com/icons?d=gallery&s=brands&v=5.0.0,5.0.1,5.0.10,5.0.11,5.0.12,5.0.13,5.0.2,5.0.3,5.0.4,5.0.5,5.0.6,5.0.7,5.0.8,5.0.9,5.1.0,5.1.1,5.2.0,5.3.0,5.3.1,5.4.0,5.4.1,5.4.2,5.5.0,5.6.0,5.6.1,5.6.3,5.9.0&m=free)
 
   - `fa` is the legacy [FA 4.7 version with its 675 icons](https://fontawesome.com/v4.7.0/icons/) but **all** of them (*and more!*) are part of FA 5.x so you should probably use the newer version above.
 
 - `ei` prefix holds [**Elusive Icons** 2.0 with its 304 icons](http://elusiveicons.com/icons/).
 
-- `mdi` prefix holds [**Material Design Icons** 3.0.39 with its 3039 icons.](https://cdn.materialdesignicons.com/3.0.39/)
+- `mdi` prefix holds [**Material Design Icons** 3.8.95 with its 3039 icons.](https://cdn.materialdesignicons.com/3.8.95/)
```
Comment (darkvertex) [inline README.md]: If you check the charmap json you produced the total icons are at 3895 now, not 3039.

**Comment by darkvertex** [comment]
And I see the PR is merged now, hmmm... New PR then I guess if you wanna fix it?

**Comment by ccordoba12** [comment]
@darkvertex, yes, please do so.

**Comment by darkvertex** [comment]
@steff456 btw, just mentioning it in case you didn't see it, in the `setup.py` there's a custom command called `update_fa5` that will update FA5 to a new version automatically with exception to updating the .rst and .md urls/totals.

If you want to save time updating next time, it works something like this:
```python setup.py update_fa5 --fa-version=5.9.0```
(The code for the command is [here](https://github.com/spyder-ide/qtawesome/blob/master/setupbase.py) if you're interested.)

It also renames the font title in the ttf, which is critical for the different styles to work properly... On that note, did you do that when you updated the files? If not, this is a broken release. :(

I'll doublecheck when I have a minute.

**Comment by ccordoba12** [comment]
> It also renames the font title in the ttf, which is critical for the different styles to work properly... On that note, did you do that when you updated the files? If not, this is a broken release. :(

I think this is already automatically tested.

**Comment by darkvertex** [comment]
> > It also renames the font title in the ttf, which is critical for the different styles to work properly... On that note, did you do that when you updated the files? If not, this is a broken release. :(
> 
> I think this is already automatically tested.

You are right, I forgot about this function: https://github.com/spyder-ide/qtawesome/blob/master/qtawesome/tests/test_qtawesome.py#L22

File: README.md (hunk 1 above)
Comment (steff456) [reply README.md]: I did not find the exact number for changing the number, so I prefered to let the same one. I will update this number, thanks!

Comment (ccordoba12) [reply README.md]: @steff456, please create a new PR to fix those minor details (here and in `docs/source/usage.rst`).

**Comment by steff456** [comment]
@darkvertex I updated FontAwesome as instructed, using the instruction you previously mentioned. I have to update the total number of icons for material design in the readme.

File: README.md (hunk 1 above)
Comment (darkvertex) [reply README.md]: > I did not find the exact number for changing the number, so I prefered to let the same one. I will update this number, thanks!

The number is easy to find if your JSON is pretty-formatted (which yours was) ... simply count the lines in your charmap json and subtract 2 to account for the opening and closing curly braces. (It's one keyword per icon after all.)

Comment (steff456) [reply README.md]: @darkvertex thanks!

**Pull request merged**